ignite/contrib/handlers/clearml_logger.py

```diff
@@ -115,15 +115,10 @@ def __init__(self, **kwargs: Any):
             from clearml import Task
             from clearml.binding.frameworks.tensorflow_bind import WeightsGradientHistHelper
         except ImportError:
-            try:
-                # Backwards-compatibility for legacy Trains SDK
-                from trains import Task
-                from trains.binding.frameworks.tensorflow_bind import WeightsGradientHistHelper
-            except ImportError:
-                raise RuntimeError(
-                    "This contrib module requires clearml to be installed. "
-                    "You may install clearml using: \n pip install clearml \n"
-                )
+            raise RuntimeError(
```

Comment:
I think we can use `ModuleNotFoundError` to replace `RuntimeError` here.

Reply:
Yes, it makes sense, thanks for the suggestion! Then it would be better to consistently raise that in all contrib handlers and classes where we check for available 3rd party library. @louis-she can you please open a new issue, I'll label it as good first issue/hacktoberfest